feat(docs): add reference to the template component (#124)
oaknational/oak-components: src/docs/designingComponents.mdx

**Call things what they are (even when they are long)**<br/>
Long names are better than ambiguous names.

## Use Component Template

> There is a componet template in the compoents library called CopyPasteThisComponent

Reviewer: ```suggestion
There is a component template in the components library called CopyPasteThisComponent
```

Reviewer: See also this comment line on the previous PR:
https://github.com/oaknational/oak-components/blob/main/src/components/CopyPasteThisComponent/CopyPasteThisComponent.stories.tsx#L7

> You can copy this compoent and replace the name with the name of your new component.

Reviewer: ```suggestion
You can copy this component and replace the name with the name of your new component.
```

> It has the index, compoent, story and test file in the folder so you just have to adapt it to you component.

Reviewer: ```suggestion
It has the index, component, story and test file in the folder so you just have to adapt it to your component.
```
